### ISSUE TITLE
Cannot take windows phone 8.1 build with X64 architecture when using this plugin

### ISSUE THREAD

**Issue by karuppub** [brody2consult/cordova-sqlite-storage#571]
I got the following error when try to build windows phone app using X64 architecuture.

The build tools for Visual Studio 2010 (v100) cannot be found. To build using the Visual Studio 2015 (v140) build tools, either click the Project menu or right-click the solution, and then select "Upgrade Solution...". 

My project target  is only windows Mobile phones 8.1 ,10 and Tablets

**Comment by brody2consult** [comment]
Use with Visual Studio 2010 is **not** supported. I think you have to get the latest version of Visual Studio 2015.

**Comment by karuppub** [comment]
Hi @brodybits 
Thanks for your reply, I am using latest version of Visual Studio 2015 with updates,
but i couldn't take build only for phone 8.1 with x64 architecture, for universal app (x64) it's working.
Without using this plugin i am able to take build for both phone8.1 and UAP with all architecture(x64,x86,arm). only after adding this plugin (cordova-sqlite-storage) i couldn't.

**Comment by brody2consult** [comment]
By "universal app (x64)" do you mean Windows 10 build for x64?

Can you elaborate what kind of a device or system you have that is "[Windows] phone 8.1 with x64 architecture"? That does not sound so common and I suspect it could be considered obsolete. Could you try building for Windows 8.1 x64 and see if it would run on your device or system?

If this doesn't work, the workaround is to use Cordova-sqlite-legacy-build-support version and build the app for the `wp8` platform.

In case you or your company would like to purchase customization please reach out to brodybits@litehelpers.net or sales@litehelpers.net. I will be available within 4-6 weeks.

**Comment by karuppub** [comment]
FYI ,My app target is only mobile platforms(windows phone 8.1 and windows mobile 10)
I couldn't run windows phone 8.1 jsproj with only x64 architecture in simulator.

For real device with arm is working
For simulator with x86 is working.
For simulator with x64 is **not working**

**Comment by brody2consult** [comment]
That makes sense, thanks. Can you post the build output for Windows Phone 8.1 x64?

**Comment by karuppub** [comment]
1>------ Skipped Build: Project: SQLite3, Configuration: Debug Win32 ------
1>Project not selected to build for this solution configuration 
2>------ Build started: Project: CordovaApp.Phone, Configuration: Debug x64 ------
2>C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\AppxPackage\Microsoft.AppXPackage.Targets(4512,5): error APPX0505: The processor architecture of your project 'x64' doesn’t match the processor architecture 'x86' of the referenced project 'SQLite3'. Change the targeted processor architectures to align between your project and its references.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 1 skipped ==========
========== Deploy: 0 succeeded, 0 failed, 0 skipped ==========

**Comment by brody2consult** [comment]
Yes: `src/windows/SQLite3-Win-RT/SQLite3/SQLite3.WindowsPhone/SQLite3.WindowsPhone.vcxproj` does not include the `x64` target platform. I plan to add this later today and will ask if you can test it.

**Comment by karuppub** [comment]
ok sure, thanks

**Comment by brody2consult** [comment]
I tried adding the `x64` target platform to the `SQLite3.WindowsPhone.vcxproj` file but when I build I get the following error output when building for `x64`:

```
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.Cpp.Platform.targets(57,5): error MSB8020: The build tools for v120_wp81 (Platform Toolset = 'v120_wp81') cannot be found. To build using the v120_wp81 build tools, please install v120_wp81 build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution".
```

I think your best bet is to use the emulator with x86 or device with ARM for testing. I don't think Windows Phone 8.1 was really meant for 64-bit builds anyway. I will document this when I get a chance.

If you or anyone else can get this working a pull request is welcome for consideration.

**Comment by karuppub** [comment]
Okay thanks!!!
